alchemy/src/cloudflare/bun-spa/bun-spa.ts

```diff
@@ -86,7 +86,7 @@ export async function BunSPA<B extends Bindings>(
     },
     build: spreadBuildProps(
       props,
-      `bun build '${frontendPaths.join("' '")}' --target=browser --minify --define:process.env.NODE_ENV='"${nodeEnv}"' --env='BUN_PUBLIC_* --outdir ${outDir}`,
+      `bun build '${frontendPaths.join("' '")}' --target=browser --minify --define:process.env.NODE_ENV='"${nodeEnv}"' --env='BUN_PUBLIC_*' --outdir ${outDir}`,
```

Comment:
I just fixed this :) https://github.com/alchemy-run/alchemy/commit/b03ce2423d410021055de9e81153137bab9d93b1